src/cmd/bootstrap.ts

```diff
@@ -208,6 +209,11 @@ const bootstrapValues = async (): Promise<void> => {
   await copyBasicFiles()
 
   const originalValues = await processValues()
+  // exit early if isCli and ENV_DIR was empty, and let the user provide valide values first:
+  if (!originalValues) {
+    debug.log('A new values repo has been created. For next steps follow otomi.io/docs.')
```

Comment:
```suggestion
    debug.log('A new values repo has been created successfully. For next steps, follow otomi.io/docs, since Otomi requires additional configuration to get started.')
```